### DISCUSSION TITLE
LED Status Anzeige

### DISCUSSION THREAD

**Discussion by boctus** [tbnobody/OpenDTU#1346] [Ideas]
Hallo Leute,
ich habe im Forum danach gesucht, aber scheinbar hat noch keiner die Frage gestellt:

An alle Experten:
Ist es möglich in der DTU die Status-Anzeige des Wechselrichters ab zu bilden. Ich meine damit die LED-Anzeige im Gehäuse des Wechselrichters. Die Blinkt alle 10 sec. wenn alles gut ist, aber in anderer Farbe oder intensität bei einem Fehler. Da meine WR auf dem Dach unter den Panelen hängen, wäre dies sinnvoll in der openDTU zu sehen.
Victron hat soetwas auch in der APP für seine z.B. Multiplus.

Viele Grüße
Martin

**Comment by Tiese** [comment]
Hallo boctus,

vielleicht geht es ja auch so: [https://github.com/tbnobody/OpenDTU/discussions/1338](https://github.com/tbnobody/OpenDTU/discussions/1338)
Da blinkt das Display, wenn 0W erzeugt wird - auch, wenn z. B. die Netzspannung ausfällt ;-)
